sql/mysqld.cc

```diff
@@ -4172,6 +4172,13 @@ int init_common_variables()
   */
   global_system_variables.time_zone= my_tz_SYSTEM;
 
+#ifdef WITH_WSREP
```

Comment:
Please document all your code-snippet. It is difficult to reverse engineer each snippet + time consuming. Defeat purpose of multiple developers working.